video/out/wayland_common.c

```diff
@@ -1825,6 +1851,14 @@ int vo_wayland_control(struct vo *vo, int *events, int request, void *arg)
         }
         return VO_TRUE;
     }
+    case VOCTRL_IS_IMAGE: {
+#if HAVE_WAYLAND_PROTOCOLS_1_27
+        wl->current_content_type = (bool)arg ? WP_CONTENT_TYPE_V1_TYPE_PHOTO :
+                                               WP_CONTENT_TYPE_V1_TYPE_VIDEO;
+        set_content_type(wl);
+#endif
```

Comment:
Pedantic: Should we send NONE if nothing is playing or we're in a configuration where there is audio with no image but the user has set it up so the window is shown?

Reply:
That's a good question. With the current code, it wouldn't send anything in that case (same as `NONE`). However, if the previous item in the playlist sent something (`VIDEO` or `PHOTO`) then it would remain unchanged. I suppose it's worth handling this case to make sure it always sends `NONE`.